src/woo_publications/metadata/update_informatie_category.py

```diff
@@ -0,0 +1,83 @@
+from io import StringIO
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management import call_command
+
+import requests
+from glom import PathAccessError, T, glom
+
+from woo_publications.metadata.constants import InformationCategoryOrigins
+from woo_publications.metadata.models import InformationCategory
+
+SPEC = {
+    "naam": T["http://www.w3.org/2004/02/skos/core#prefLabel"][0]["@value"],
+    "naam_meervoud": T[
+        "https://identifier.overheid.nl/tooi/def/ont/prefLabelVoorGroepen"
+    ][0]["@value"],
+    "definitie": T["http://www.w3.org/2004/02/skos/core#definition"][0]["@value"],
+    "order": T["http://www.w3.org/ns/shacl#order"][0]["@value"],
+}
+
+
+class InformatieCategoryWaardenlijstError(Exception):
+    pass
+
+
+def update_informatie_category(file_path: str):
+    if not file_path:
+        file_path = str(
+            Path(
+                settings.BASE_DIR
+                / "src"
+                / "woo_publications"
+                / "fixtures"
+                / "informatie_category.json"
+            )
+        )
+
+    response = requests.get(
+        "https://repository.officiele-overheidspublicaties.nl/waardelijsten/scw_woo_informatiecategorieen/3/json/scw_woo_informatiecategorieen_3.json"
+    )
+
+    try:
+        response.raise_for_status()
+    except requests.exceptions.ConnectionError as err:
+        raise InformatieCategoryWaardenlijstError(
+            "Could not connect with url."
+        ) from err
```

Comment:
this will never raise ConnectionError`, since that would be raised already in the `requests.get` call.

Probably better to:

```py
try:
    response = requests.get(...)
except requests.RequestsError as err:
    raise InformatieCategoryWaardenlijstError("Could not retrieve the value list data.") from err

try:
    response.raise_for_status()
except requests.RequestError:
    raise InformatieCategoryWaardenlijstError(f"Got an unexpected response status code when retrieving the value list data: {response.status_code}.") from err